third_party/terraform/website/docs/guides/version_3_upgrade.html.markdown

```diff
@@ -397,6 +398,45 @@ required on the `google_compute_router_peer.advertised_ip_ranges` block.
 In an attempt to avoid allowing empty blocks in config files, `raw_key` is now
 required on the `google_compute_snapshot.source_disk_encryption_key` block.
 
+## Resource: `google_compute_subnetwork`
+
+### `enable_flow_logs` is now removed
+
+`enable_flow_logs` has been removed and should be replaced by the `log_config` block with configurations
```

Comment:
Probably worth saying "Enablement of flow logs is now controlled by whether `log_config` is defined or not instead of by the `enable_flow_logs` variable.", as well as noting that users who has `enable_flow_logs = false` only need to remove it.